harvester_e2e_tests/apis/test_settings.py

```diff
@@ -162,3 +162,35 @@ def test_invalid_S3(self, api_client):
             f"S3 backup-target should check key/secret/bucket/region"
             f"API Status({code}): {data}"
         )
+
+
+@pytest.mark.p0
+@pytest.mark.negative
+@pytest.mark.settings
+@pytest.mark.settings
+@pytest.mark.skip_version_before('v1.3.1')
```

Comment:
This issue seems fixed in `v1.4.0`.
In current design, `skip_version_before(target_ver)` actually includes `target_ver`. (See _fixtures/api_client.py_)
So these will still be picked with `v1.3.1` cluster.
![image](https://github.com/user-attachments/assets/07a3ba97-3b84-494c-b793-6381f4244856)


Suggest to use `skip_version_if` instead, it's more explicit, e.g.
`@pytest.mark.skip_version_if("> v1.2.0", "<= v1.4.0", reason="Issue#4293 fix after v1.4.0")`

Reply:
That's a great mention :smile: :+1:  - I've implemented that decorator on the 1 negative test that will fail.
The other, positive test, I've just left the prior decorator, as that should always pass.

Reply:
Actually, it's fixed in `v1.3-head` - so it will be fixed/working in v1.3.2 - so I just set the decorator to that - and updated that case :smile: